vue/components/ui/organisms/entity-shower/entity-shower.vue

```diff
@@ -28,50 +28,91 @@ export const EntityShower = {
     name: "entity-shower",
     mixins: [partMixin],
     props: {
+        /**
+         * The entity name
+         */
         name: {
             type: String,
             required: true
         },
+        /**
+         * Entity fields that are going to be shown

```

Comment:
Be more descriptive, from the docs I can't understand what must be passed here without checking the implementation.

Reply:
In general this applies to most docs. Always think to yourself that if a doc is enough to use the prop without checking the rest of the code.

Reply:
Should I also add the prop type in the documentation?
I tried to follow the comments in `pickers` and `configurator`

Reply:
No need, but you can add an example (e.g.: [{ name: ... , ... }])